src/main/java/helper/GoogleUserMapper.java

```diff
@@ -0,0 +1,33 @@
+package helper;
+
+import java.io.ByteArrayOutputStream;
+import java.io.IOException;
+import java.io.InputStream;
+import java.net.URL;
+import java.net.URLConnection;
+import java.util.Base64;
+
+public class GoogleUserMapper {
+    public String getUsernameFromEmail(String email) {
+        int usernameEndsAt = email.indexOf('@');
+        if(usernameEndsAt == -1) {
+            return null;
```

Comment:
Why return null here?
Shouldn't it be an error here for an email to not have the character `@` in it?
Perhaps we can throw `IllegalArgumentException`.

Reply:
Yes, Done.